src/server.js

```diff
@@ -1,15 +1,13 @@
-const http = require('http');
-const url = require('url');
+const http = require("http");
+const url = require("url");
 
-const morgan = require('morgan');
-const router = require('./routes/router');
+const morgan = require("morgan");
+const router = require("./routes/router");
 
-const logger = morgan('combined');
+const logger = morgan("combined");
 
 const startServer = port => {
-
   const server = http.createServer((request, response) => {
-
     // Get route from the request
     const parsedUrl = url.parse(request.url);
 
```

Comment:
Also could you add handler with log for listen function? It's not obvious, when and on which port your server ready to accept requests after script start